package.json

```diff
@@ -12,7 +12,7 @@
     "format:package_json": "lerna exec -- prettier-package-json",
     "postinstall": "husky",
     "lint": "npm run lint:js && npm run lint:md",
-    "lint:js": "eslint .storybook/ packages/ utils/ --ext js,mjs,ts,tsx --max-warnings 0",
+    "lint:js": "eslint . --max-warnings 0",
```

Comment:
Want to call attention to this to check if perhaps I'm mishandling an issue I ran into. 

Originally, I retained the same command (sans `--ext`) and specified the following in `eslint.config.mjs`:

```
{
  ignores: ["!.storybook", "packages/**/dist"]
},
```

However I kept getting this error:

```
Oops! Something went wrong! :(

ESLint: 9.7.0

You are linting "packages/", but all of the files matching the glob pattern "packages/" are ignored.

If you don't want to lint these files, remove the pattern "packages/" from the list of arguments passed to ESLint.
```

I double checked and didn't see any reason `packages` would be ignored here (given CLI args and config as written above). I tried a few variants as well: `packages/*/dist/`, `packages/*/dist/*`, etc.

I figured maybe multiple inputs to the default CLI argument might be the cause, but then I noticed ESLint's [own examples](https://eslint.org/docs/latest/use/command-line-interface#run-the-cli) show multiple inputs to the default argument of the `eslint` CLI.

Again, open to input if this workaround seems undesirable.

Reply:
I could be wrong but `ignores: ['**/dist']` should exclude the intended folders.

Reply:
What do you mean? The intent is to exclude all `dist` folders. Or are you saying this is doing the opposite?